src/libraries/System.Private.CoreLib/src/System/Guid.cs

```diff
@@ -60,7 +60,7 @@ public Guid(ReadOnlySpan<byte> b)
             }
 
             // slower path for BigEndian:
-            _k = b[15];  // hoist bounds checks
+            _k = b[15]; // hoist bounds checks
```

Comment:
nit: revert the whitespace change

Reply:
That must have been an accident on my part, I didn't notice the double space in the original code. Addressed in https://github.com/dotnet/runtime/pull/78446/commits/8c2907ebb3d3988e01d00ece3060c406e86739f2.